homeassistant/components/ezviz/switch.py

```diff
@@ -55,7 +55,7 @@ def __init__(
     @property
     def is_on(self) -> bool:
         """Return the state of the switch."""
-        return self.data["switches"][self._name]
+        return self.data["switches"].get(self._name, False)
```

Comment:
If the key doesn't exist, should it not simply return `None`?
I don't know the integration, but I think it would make more sense to set the return type as `bool | None`

```suggestion
        return self.data["switches"].get(self._name)
```